client/src/modules/reports/generate/cash_report/cash_report.html

```diff
@@ -0,0 +1,80 @@
+  on-clear-callback="ReportConfigCtrl.clearPreview()"
+  on-save-callback="ReportConfigCtrl.requestSaveAs()">
+</bh-report-preview>
+
+<div ng-show="!ReportConfigCtrl.previewGenerated">
+  <div class="row">
+    <div class="col-md-12">
+      <h3 class="text-capitalize" translate>REPORT.CASH_REPORT.TITLE</h3>
+      <p class="text-info" translate>REPORT.CASH_REPORT.DESCRIPTION</p>
+    </div>
+  </div>
+
+  <div class="row" style="margin-top : 10px">
+    <div class="col-md-6">
+      <div class="panel panel-default">
+        <div class="panel-heading">
+          <span translate>REPORT.UTIL.OPTIONS</span>
+        </div>
+
+        <div class="panel-body">
+          <form name="ConfigForm" bh-submit="ReportConfigCtrl.preview(ConfigForm)" novalidate>
+            
+            <!-- report type selection  -->
+            <div 
+                class="form-group"
+                ng-class="{ 'has-error' : ConfigForm.$submitted && ConfigForm.type.$invalid }">
+                <label class="control-label">
+                    <i class="fa fa-briefcase"></i> {{ "FORM.SELECT.REPORT_TYPE" | translate }}
+                </label>
+                <select
+                    class="form-control"
+                    name="type"
+                    ng-model="ReportConfigCtrl.reportDetails.type"
+                    ng-options="type.id as (type.label | translate) for type in ReportConfigCtrl.reportTypes track by type.id"
+                    required>
+                    <option value="" disabled>{{ "FORM.SELECT.REPORT_TYPE" | translate }}<option>
+                </select>
+                <div class="help-block" ng-messages="ConfigForm.type.$error" ng-show="ConfigForm.$submitted">
+                    <div ng-messages-include="modules/templates/messages.tmpl.html"></div>
+                </div>
+            </div>
+
+            <!-- cashbox selection  -->
+            <div 
+                class="form-group"
+                ng-class="{ 'has-error' : ConfigForm.$submitted && ConfigForm.cashbox.$invalid }">
+                <label class="control-label">
+                    <span class="fa fa-briefcase"></span> {{ 'FORM.SELECT.CASHBOX' | translate }}
+                </label>
+                <select
+                    class="form-control"
+                    name="cashbox"
+                    ng-model="ReportConfigCtrl.reportDetails.cashbox"
+                    ng-options="cash as cash.hrlabel for cash in ReportConfigCtrl.cashboxes track by cash.id"
+                    required>
+                    <option value="" disabled>{{ "FORM.SELECT.CASHBOX" | translate }}</option>
+                </select>
+                <div class="help-block" ng-messages="ConfigForm.cashbox.$error" ng-show="ConfigForm.$submitted">
+                <div ng-messages-include="modules/templates/messages.tmpl.html"></div>
+                </div>
+            </div>
+
+            <!-- Date interval  -->
+            <bh-date-interval
```

Comment:
It would be really nice if this had a default set of dates.  It may just be on a slow internet connection, but when I load this on `staging.bhi.ma`, it doesn't have default dates.